OracleDatabase/RAC/OracleRealApplicationClusters/dockerfiles/18.3.0/MultiNodeInstall.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/python
+#!/usr/bin/env python
+
+###########################################################################################################
+
+
+# stig.py
+#
+# Copyright (c) 2012, 2018, Oracle and/or its affiliates. All rights reserved.
```

Comment:
This file needs at least the 1-line UPL header.

Reply:
Added the details.